opus/import/obs_bundle_cassini_iss_fring_mosaics_rsfrench2025.py

```
@@ -0,0 +1,316 @@
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+
+    #######################
+    ### HELPER FUNTIONS ###
+    #######################
+
+    def _camera(self):
+        return self._index_col('min_image_name')[-1].upper()
+
+    def _circumference(self):
```

Ruff (0.15.10)

[warning] Missing return type annotation for private function `_circumference`

(ANN202)